server/index.js

```diff
@@ -168,13 +168,11 @@ app.get("/*/contributors.txt", async (req, res) => {
     return res.status(404).send(`Document not found by URL (${url})`);
   }
   const { doc: builtDocument } = await buildDocument(document);
-  if (document.metadata.contributors || !document.isArchive) {
+  if (document.metadata.contributors) {
```

Comment:
Ditto here. Since all content is now non-archived, `!document.isArchive` would always be true, so this condition is always satisifed and therefore we should always run the code right?